src/infra/postgres/postgres.go

```diff
@@ -16,4 +16,10 @@ type Client interface {
 	DeletePK(ctx context.Context, model ...interface{}) error
 	ForceDeletePK(ctx context.Context, model ...interface{}) error
 	RunInTransaction(ctx context.Context, persist func(client Client) error) error
+
+	SelectMany(ctx context.Context, model interface{}, dst interface{}, condition string, params ...interface{}) error
+
+	Ping(ctx context.Context) error
```

Comment:
Same comment about this should not appear here as it's not used by the app.

Reply:
Finally I removed it with the tests not using dockertest anymore.